src/QueueReceiver.Infrastructure/Data/DbContextSettings.cs

```diff
@@ -3,5 +3,8 @@
     public class DbContextSettings
```

Comment:
I think we should move the  PersonProjectCreatedUsername to somewhere else. As it doesn't belong in this type or class or namespace anymore.

Reply:
DbContextSettings rewritten to PersonCreatedByCache